packages/applications/src/views/Home.jsx

```diff
@@ -0,0 +1,98 @@
+                      </li>
+                      <li className='my-4'>
+                        It's OK if you don't understand what creating your own keys means, or how to keep them safe. We're here to <strong>learn together</strong>. Follow along as best you can, and continue to the application form when prompted. Though you are the only one who can see your keys, we can help you make new ones while keeping your data safe, private, and under your control.
+                      </li>
+                      <li className='my-4'>
+                        Once you have created your keys, you will be redirected to an application form. Respond to the questions as best you can and click 'Submit' once you are happy.
+                      </li>
+                      <li className='my-4'>
+                        Come back here whenever you like, login with your new account, and check your status. We review applications on a rolling basis and will update you here and via email about any changes.
+                      </li>
+                    </ol>
+                    <p className='my-12'>
+                      Even if you don't get in to Kernel immediately, you are free to read <a className='text-blue-600 visited:text-purple-600' href='https://kernel.community/en/learn/' target='_blank' rel='noreferrer'>the syllabus</a>, learn from<a className='text-blue-600 visited:text-purple-600' href='https://kernel.community/en/build' target='_blank' rel='noreferrer'> everything we build</a>, watch <a className='text-blue-600 visited:text-purple-600' href='https://www.youtube.com/channel/UC2kUaSgR0L-uzGkNsOxSxzw' target='_blank' rel='noreferrer'>all our recordings</a>, and use or adapt <a className='text-blue-600 visited:text-purple-600' href='https://github.com/kernel-community/' target='_blank' rel='noreferrer'>all our code </a>. Even better: creating your own, personal key-pair enables you to use many of our best tools right now. Your new account will help you learn how to mint tokens, deploy your own contracts, decode transactions on a blockchain and much more. You do not need our permission to use it, and you can start right now...
+                    </p>
+                  </div>
+                  <div className='text-center mx-auto shadow-lg rounded-lg bg-gray-300 border-0 lg:w-80'>
+                    <button
+                      className='bg-gray-900 text-white active:bg-gray-700 text-sm font-bold uppercase px-6 py-3 rounded shadow hover:shadow-lg outline-none focus:outline-none mr-1 mb-1 w-full'
+                      onClick={handleLogin}
+                      type='button'
+                    >
+                      Create your keys
```

Comment:
I think if you are a returning user, it might not be obvious that you should click on the same button.

The `walletLogin` function handles both cases as a convenience doing the right depending on whether you have an existing wallet, expired credentials etc.

We should choose a neutral phrase that reflects that fact such as `Sign in with Kernel`.

Alternatively, you can just have two buttons one with `Create your keys` and one with `Login with your Kernel Wallet` and bot pointing at the same `handleLogin` function.

Reply:
Changed the flow quite a bit here, part of which accommodates this.